src/containers/Workspace.js

```diff
@@ -20,8 +21,26 @@ const mapStateToProps = state => (
   }
 );
 
+
+/**
+ * @param theme
+ */
+const styles = theme => ({
+  workspaceViewport: {
+    background: (theme.palette.type === 'light') ? theme.palette.grey[100] : theme.palette.grey[700],
```

Comment:
Instead of calculating this in a couple places, maybe we can explicitly add the value to the theme?

Reply:
Can you say more about that? I'm not sure how to do that without still having to check what theme is currently selected

Reply:
I'm thinking something like akin to how we're using `theme.palette.darkened` or `.lightened` currently:

- https://github.com/ProjectMirador/mirador/blob/master/src/config/settings.js
- https://github.com/ProjectMirador/mirador/blob/master/src/containers/CompanionArea.js#L41

Reply:
Ah.. I was hoping we could eventually remove those at some point (I'm not a fan of them.. )

Reply:
Yeah, I'm not sure they're entirely idiomatic, but it has the benefit of being externally theme-able instead of hardcoding colors in our components.

Reply:
true.. what if we added something like `hue` which would allow adopters to pass a mui color HUE that was then used? https://material-ui.com/style/color/

```json
    palette: {
      type: 'light', // dark also available
      hue: 'grey'
      primary: {
        main: '#1967d2',
      },
      secondary: {
        main: '#ffa224',
      },
      grey: {} // must be defined for `hue`
```

That way an adopter could then use any color hue w/ varying shades

Reply:
meh.. now I don't like that proposal. I'm going to do something similar to your first suggestion :)

Reply:
@cbeer I've updated using your suggestion